public/css/home.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+
+:root{
+    --food-slider-bg:#171A29;
+    --food-top-brands:#F8F8F8;
+    --card-bg:#F5F5F5;
+    --border-black-op: rgba(0,0,0,0.1);
+}
+.nav-zindex{
+    z-index: 999;
+}
+.container{
+    width: 95%;
+    margin: auto;
+}
+
+.navbar-bg{
+  padding: 1rem 1rem 0;
+  border-bottom: 1px solid var(--border-black-op);
+
+}
+
```

Comment:
fix : but then why 2 spaces here ? either leave 1 or 2 spaces , and follow that everywhere.